components/admin/lessons/AdminModuleInputs.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useState } from 'react'
+import {
+  AddModuleMutation,
+  UpdateModuleMutation,
+  useAddModuleMutation,
+  useUpdateModuleMutation
+} from '../../../graphql'
+import { formChange } from '../../../helpers/formChange'
+import { FormCard, MD_INPUT, TextField } from '../../FormCard'
+import { AlertFillIcon, CheckCircleIcon } from '@primer/octicons-react'
+import styles from '../../../scss/adminModuleInputs.module.scss'
+import { Spinner } from 'react-bootstrap'
+import { get } from 'lodash'
+import {
+  ApolloError,
+  OperationVariables,
+  ApolloQueryResult
+} from '@apollo/client'
+
+type Module = { id: number; name: string; content: string }
+
+type Props = {
+  lessonId: number
+  refetch: (variables?: Partial<OperationVariables> | undefined) => Promise<
```

Comment:
Is `| undefined` needed here? Since variables is optional

Reply:
No, it isn't.